bme_test_suite.yaml

```diff
@@ -0,0 +1,37 @@
+---
+- hosts: deploy_node
+  name: Install and run tests
+  vars:
+    tempest_root: /opt/tempest_untagged
+    test_name: persistent-resources-tests
+    test_type: persistent-clean
+
+  tasks:
+    - name: "Clone git repo {{ test_name }} into {{ tempest_root }}/{{ test_name }}"
+      when: test_name == "persistent-resources-tests"
+      command: /usr/local/bin/ansible utility_all -i /opt/openstack-ansible/playbooks/inventory -m shell -a "git clone https://github.com/osic/{{ test_name }}.git {{ tempest_root }}/{{ test_name }}"
+
+    - name: "Install {{ test_name }} into {{ tempest_root }}/{{ test_name }}"
+      when: test_name != "tempest"
+      command: /usr/local/bin/ansible utility_all -i /opt/openstack-ansible/playbooks/inventory -m shell -a "pip install {{ tempest_root}}/{{ test_name }}"
```

Comment:
I guess at this point you would install the persist-resources-tests if those were the ones selected by the used in the args... if that is so, why there is a separate playbook for installing persistent resources tests?